docs/docs/gatsby-config.md

```diff
@@ -115,6 +115,8 @@ module.exports = {
 }
 ```
 
+You may need to install the appropriate file transformer (in this case [YAML](https://www.gatsbyjs.org/packages/gatsby-transformer-yaml/)) and set up [gatsby-source-filesystem](https://www.gatsbyjs.org/packages/gatsby-source-filesystem/) properly for Gatsby to pick up the mapping files. This applies to other file types later mentioned in this segment as well.
```

Comment:
```suggestion
You may need to install the appropriate file transformer (in this case [YAML](/packages/gatsby-transformer-yaml/)) and set up [gatsby-source-filesystem](/packages/gatsby-source-filesystem/) properly for Gatsby to pick up the mapping files. This applies to other file types later mentioned in this segment as well.
```
We should keep these URLs relative to the docs site. But that's it! Looking great!

Reply:
@marcysutton updated :)))))